views/github-create-branch-options.hbs

```diff
@@ -57,6 +58,13 @@
     </form>
   </section>
 
+  <div class="gitHubCreateBranchOptions__loading">
+    {{> loading-screen
+        title="Connecting to GitHub"
+        message="Please close this window when you are done creating a branch!"
```

Comment:
Not sure if we need this one?
message="Please close this window when you are done creating a branch!"

Reply:
An optional text, so that the users would know to close the iframe window if it is still opening somehow.

Reply:
mmm fair, 
can we make it not break onto a new line with styles or slightly shorter message?

Reply:
Updated the text to `Please close this window after creating a branch.`